public/server/db/btaim.json

```
@@ -0,0 +1,4 @@
+{
+    "definition": "Be that as it may"
```

The first letter of each word should be a capital letter,

```suggestion
    "definition": "Be That As It May"
```